package.json

```diff
@@ -13,9 +13,15 @@
   "license": "ISC",
   "dependencies": {
     "@cardano-ogmios/client": "^4.2.1",
+    "@emurgo/cardano-serialization-lib-nodejs": "^9.1.2",
```

Comment:
I've forked sundae's fork of CSL, so we should switch this once I've uploaded the NPM packages. According to @klntsky, their APIs have diverged, hopefully there won't be any issues though

Reply:
ok